backend/Tasque.Core.WebAPI/Program.cs

```diff
@@ -28,7 +28,7 @@
         b => b.MigrationsAssembly(typeof(DataContext).Assembly.FullName))
         .EnableDetailedErrors());
 
-builder.WebHost.UseUrls("http://*:5000");
+//builder.WebHost.UseUrls("http://*:5000");
```

Comment:
Please, don't commit this line or move it to appsettings, deploy expects to use defined port